.travis.yml

```diff
@@ -6,9 +6,17 @@ sudo: false
 cache:
   directories:
   - node_modules
+
 before_install:
 # Symlink closure library
 - ln -s $(npm root)/google-closure-library ../closure-library
+
+before_script:
+  - export DISPLAY=:99.0
+  - if [ "${TRAVIS_OS_NAME}" == "linux" ]; then ( scripts/setup_linux_env.sh  ) fi
```

Comment:
Note that these scripts are now under tests/scripts/ in Blockly on develop, but not yet on master.  You probably want them under tests/scripts here as well.

Reply:
Fixed 
d1ee51e